blueocean-personalization/.storybook/config.js

```diff
@@ -0,0 +1,7 @@
+import { configure } from '@kadira/storybook';
```

Comment:
Could all of this somehow be automated with js-modules or another devDependency?